docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>The run/processing function can have more than one input or output. The function with <code>n</code>  ... </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>